Makefile

```diff
@@ -163,3 +163,15 @@ name-collision:
 
 composer-dependency-analyser:
 	php vendor/bin/composer-dependency-analyser --config build/composer-dependency-analyser.php
+
+.PHONY: infection
+infection:
+	git clone https://github.com/phpstan/build-infection.git || true
+	git -C build-infection fetch origin && git -C build-infection reset --hard origin/1.x
+	composer install --working-dir build-infection --no-interaction --no-progress
+	php build-infection/bin/infection-config.php --source-directory='build/PHPStan/Build'> infection.json5
+	version=$(shell jq -r '.packages[] | select(.name == "infection/infection") | .version' build-infection/composer.lock); \
+	wget https://github.com/infection/infection/releases/download/$$version/infection.phar --no-clobber;
```

Comment:
You don't need to download infection.phar, you can run infection from build-infection/vendor/bin/infection